package.json

```diff
@@ -8,12 +8,12 @@
   "dependencies": {
     "@fastify/accept-negotiator": "^1.1.0",
     "fastify-plugin": "^4.5.0",
-    "into-stream": "^6.0.0",
     "mime-db": "^1.52.0",
     "minipass": "^7.0.2",
     "peek-stream": "^1.1.3",
     "pump": "^3.0.0",
-    "pumpify": "^2.0.1"
+    "pumpify": "^2.0.1",
+    "readable-stream": "^4.5.2"
```

Comment:
Not blocking but I just don't see the reason for adding _readable-stream_ **if the test suit is passing without it**, I remember PRs that removed that package being merged as well

Also I get the stability argument but with the same point of view one could say that any part of node's API can change in a breaking manner between major versions, no? Then why not install every single library as a package?

Maybe _readable-stream_ is special, so just asking to learn

Reply:
We can ensure that bugs in streams not solved in older node versions, like in node14, are not affecting our implementation.